tools/config/index.ts

```diff
@@ -294,6 +292,8 @@ if (options.generateDeps) {
  * Format all files.
  */
 if (options.fix) {
+  console.log('\nGenerating updated package-lock file...\n');
+  execSync('npm install', { stdio: 'inherit' });
```

Comment:
Here we're applying change sin the `config:update`.

IMHO its counterpart, the `config:check` ideally it should fail when the existing-git-comitted `pacakge-lock.json` contents are different than those generated by new run of `npm install`. It would help spot that the author of the changes in the PR forgot to commit their changes in `package-lock.json`. And then such a PR would fail in CI.

In other words, I'd suggest that the `config:check` logic could do the following:
```ts
const gitPackageLock= execSync(`git show HEAD:package-lock.json`, { encoding: 'utf8' });
execSync('npm install', { stdio: 'inherit' });
const newPackageLock = readFileSync('package-lock.json', 'utf8');

if (gitPackageLock !== newPackageLock) {
  // throw error...
}
```